docs/automations/integrations/jira/update-jira/README.md

```diff
@@ -0,0 +1,28 @@
+# Automatically Update Jira Tickets from PRs
+
+Automatically update Jira tickets based on PR status.
+
+[start:example]
+
+!!! info "Conditions (All Must be True):"
+    * The PR contains a reference to a Jira ticket in the title or description
+
+!!! example "Automation Actions:"
+    * Send an HTTP request to a Jira endpoint that updates the ticket status.
+
+```yaml
```

Comment:
This example should be imported from the associated .cm file on this PR rather than including the full text here.